articles/automated-fake-news-detection/index.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/debug-node-devtools/
+title: Introduction to Automated Fake News Detection
+description: 
+author: 
+date: 2020-09-10T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/automated-fake-news-detection/hero.jpg
+    alt: Devtools example image Node.js debugger
+---
+Misinformation presents a huge challenge in today’s online society. As a result, there have been many attempts to identify and classify misinformation. Specifically, in social networking sites, blogs as well online newspapers.
+<!--more-->
+
+Since this is a very broad research area, in this article we’ll look at:
+· What is fake news and why you should care.
+· Manual vs automated fake news detection efforts.
+· Brief introduction to machine learning and deep learning techniques to fake news detection.
+· Bonus: BERT.
```

Comment:
These are all on one line. I think you're missing a return or something.

Reply:
Sorting out the format for all such occurrences right now